cleanlab_studio/errors.py

```diff
@@ -0,0 +1,10 @@
+class InvalidDatasetError(RuntimeError):
```

Comment:
Subclass `ValueError` instead? It's more specific and makes sense in this context.